### PR TITLE
380 ionic cli package

### PR DESCRIPTION
**Pull request by ManuelFeller**
Pull request for Issue #380 
- updated Ionic CLI package
- removed yarn usage to use npm consistently 
- removed `package.json` from `files` dir to use the generated default one
- removed `karma.conf.js` from `files` dir to use the generated one
- added replacement scripting for `karma.conf.js` in `init.sh` similar to the Angular Quickstarter

### CODE REVIEW

File: fe-ionic/init.sh [hunk 1]
```diff
@@ -18,11 +18,56 @@ while [[ "$#" > 0 ]]; do case $1 in
 esac; shift; done
 
 echo "generate project"
+<<<<<<< HEAD
```
Comment (clemensutschig) [inline fe-ionic/init.sh:21]: @ManuelFeller  - merge conflict? ....

Comment (ManuelFeller) [reply fe-ionic/init.sh:21]: checking right now, had solved that already locally (I thought)

Comment (ManuelFeller) [reply fe-ionic/init.sh]: OK, no longer in there now, added some explanations about what is happening

File: fe-ionic/init.sh [hunk 2]
```diff
@@ -18,11 +18,58 @@ while [[ "$#" > 0 ]]; do case $1 in
 esac; shift; done
 
 echo "generate project"
+# create temp-dir since non-interactive ionic-cli can not generate into existing directories
 mkdir start_$COMPONENT
 cd start_$COMPONENT
-ionic start $COMPONENT blank --type=angular --no-deps --no-git
+
+# creating project
+ionic start $COMPONENT blank --type=angular --no-deps --no-git --no-interactive
```
Comment (cschweikert) [inline fe-ionic/init.sh:26]: ``--no-interactive`` is not needed. Since ``CI=true`` it has the same behavior.

File: fe-ionic/init.sh [hunk 3]
```diff
@@ -18,11 +18,58 @@ while [[ "$#" > 0 ]]; do case $1 in
...
+
+# move generated project to the intended directory
 mv $COMPONENT/ ..
+# switch to component directory
 cd ../$COMPONENT
+# remove empty temp-dir
+rm ../start_$COMPONENT
+
+echo "Change test setup to single run in karma.conf.js"
```
Comment (cschweikert) [inline fe-ionic/init.sh:35]: ``fe-ionic/files/package.json`` was removed. On the bottom of this ``init.sh`` is still a ``rm ./package.json``. After this there wouldn't be any package.json left...

beside that the ``fe-ionic/files/package.json`` had some finetuning on the tasks like added parameters for generating code coverage analysis data:
```
"test": "ng test --code-coverage --reporters=junit",
```

File: fe-ionic/init.sh (hunk 2 above)
Comment (ManuelFeller) [reply fe-ionic/init.sh:26]: Idea behind is to have same behaviour reproducible in your local system without having anything to remember. But can remove if wanted...

File: fe-ionic/init.sh (hunk 3 above)
Comment (ManuelFeller) [reply fe-ionic/init.sh:35]: Done as requested

**Pull request merged**